### PR TITLE
Introduce install command

### LINKED ISSUE #107: Build install command
We need an install command which asks for username and password and email for the first user.

### PR DESCRIPTION
**Pull request by bambamboole**
This PR resolves #107

### CODE REVIEW

File: resources/stubs/CmsServiceProvider.stub [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        $router->registerPagesRoutes();
+        $router->registerPostsRoutes();
+    }
+
+    public function register()
+    {
+//        This way we can enable a custom theme
+//        $this->app->singleton(Theme::class, function (){
+//            return new YourCustomTheme();
+//        });
```
Comment (slashplusweb) [inline resources/stubs/CmsServiceProvider.stub:22]: What is this? soomething for later or forgotten code?

File: src/Core/Commands/InstallCommand.php [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+            ]);
+            $this->line('User email: <info>admin@admin.com</info>');
+            $this->line('Password: <info>password</info>');
+        } else {
+            $admin = User::query()->create([
+                'name' => $this->ask('Name for the first admin user?'),
+                'email' => $this->ask('Email for the first admin user?'),
+                'password' => $this->secret('Password for the first admin user?'),
+            ]);
+        }
```
Comment (slashplusweb) [inline src/Core/Commands/InstallCommand.php:96]: Please apply defaults when no admin string will be provided. (this should also be mentioned when prompting the user to input something)

File: resources/stubs/CmsServiceProvider.stub (hunk 1 above)
Comment (bambamboole) [reply resources/stubs/CmsServiceProvider.stub:22]: Its just a commented snippet showing how to use a custom theme

File: src/Core/Commands/InstallCommand.php (hunk 2 above)
Comment (bambamboole) [reply src/Core/Commands/InstallCommand.php]: defaults are done

**Pull request merged**